pt-BR/debugging_rails_applications.md

```diff
@@ -1,16 +1,16 @@
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Debugging Rails Applications
+Debugando Aplicações Rails
 ============================
 
-This guide introduces techniques for debugging Ruby on Rails applications.
+Esse guia introduz técnicas de debugação para aplicações de Ruby on Rails
```

Comment:
Não sei se existe alguma orientação quanto a tradução de alguns palavras, mas acredito  que a palavra `debugging` poderia ficar sem tradução. Qual sua opinião @HenriqueMorato ?

Reply:
Eu já ouvi gente usando debugar, mas também não tenho certeza. Talvez deixar debugging fosse melhor mesmo.

Reply:
Acho que podia ser algo como "técnicas de *debug*" colocando termos em inglês em itálico. O que vocês acham?

Reply:
Ou usar o termo `depurar` que é o português, apesar de estranho.